app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyHomeDetailScreen.kt

```diff
@@ -0,0 +1,174 @@
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import com.google.accompanist.pager.ExperimentalPagerApi
+import com.google.accompanist.pager.HorizontalPager
+import com.kks.nimblesurveyjetpackcompose.R
+import com.kks.nimblesurveyjetpackcompose.model.Survey
+import com.kks.nimblesurveyjetpackcompose.ui.theme.BlackRussian
+import com.kks.nimblesurveyjetpackcompose.ui.theme.NeuzeitFamily
+import com.kks.nimblesurveyjetpackcompose.ui.theme.White70
+import com.kks.nimblesurveyjetpackcompose.util.TWEEN_ANIM_TIME
+import com.ramcosta.composedestinations.annotation.Destination
+
+@OptIn(ExperimentalPagerApi::class)
+@Destination
+@Composable
+fun SurveyHomeDetailScreen(survey: Survey) {
+    var currentPage by remember { mutableStateOf(0) }
+
+    Box(modifier = Modifier.fillMaxSize()) {
+        AsyncImage(
+            model = survey.coverImageUrl,
+            contentDescription = stringResource(id = R.string.survey_detail_background_image),
+            modifier = Modifier.fillMaxSize(),
+            placeholder = painterResource(id = R.drawable.ic_launcher_background),
+            contentScale = ContentScale.Crop
+        )
+        Image(
+            painter = painterResource(id = R.drawable.survey_overlay),
+            contentDescription = null,
+            modifier = Modifier.fillMaxSize(),
+            contentScale = ContentScale.Crop
+        )
+        HorizontalPager(
+            count = 1,
+            modifier = Modifier.padding(start = 20.dp, end = 20.dp, top = 108.dp),
+            userScrollEnabled = false
+        ) { page ->
+            currentPage = page
+            if (page == 0) {
+                SurveyDetailStartScreen(survey, modifier = Modifier.fillMaxWidth())
+            }
+        }
+        SurveyToolbar(
+            modifier = Modifier
+                .fillMaxWidth()
+                .align(Alignment.TopCenter)
+                .padding(top = 57.dp, start = 15.dp, end = 15.dp),
+            showBack = currentPage == 0,
+            showClose = currentPage > 0
+        )
+        StartSurveyBtn(
+            onStartSurvey = { /*TODO*/ },
+            modifier = Modifier
+                .align(Alignment.BottomEnd)
+                .padding(bottom = 54.dp, end = 20.dp)
+        )
+    }
+}
+
+@Composable
+fun StartSurveyBtn(onStartSurvey: () -> Unit, modifier: Modifier = Modifier) {
+    var showStartSurveyBtn by remember { mutableStateOf(true) }
+
+    Crossfade(
```

Comment:
Why are we crossfading this in? Surely it should fade in with the rest of the screen when navigating anyway? 🤔

Reply:
It is also crossfading in Figma too. That's why I made it like that.

Reply:
What I meant was the whole screen should fade in, not only the button 🤔

Reply:
From the animation of Figma, only button faded away and the questions scroll in like pager no?

Reply:
Ah, so the initial state is visible and then we fade out. got it